_posts/2024-07-08-geoserver-tutorials.md

```diff
@@ -0,0 +1,97 @@
+---
+author: Nima Ghasemloo
+layout: post
+title: Utilizing the Demo Section in Geoserver
+date: 2024-07-08
+categories:   
+- Tutorials
+---
+
+[GeoSpatial Techno](https://www.youtube.com/@geospatialtechno) is a startup focused on geospatial information that is providing e-learning courses to enhance the knowledge of geospatial information users, students, and other startups. The main approach of this startup is providing quality, valid specialized training in the field of geospatial information.
+
+( [YouTube](https://www.youtube.com/@geospatialtechno)
+| [LinkedIn](https://www.linkedin.com/in/geospatialtechno)
+| [Facebook](https://www.facebook.com/geospatialtechno)
+| [X](https://twitter.com/geospatialtechn)
+)
+
+----
+
+### Utilizing the Demo Section in Geoserver
+In this session we aim to provide content on how to use the **Demo menu and its modules in GeoServer** and teach the process of making different requests and getting responses from the server. If you want to access the complete tutorial, simply click on the [link](https://youtu.be/3xKKI8irlQk).
+
+[![](https://img.youtube.com/vi/3xKKI8irlQk/0.jpg)](https://www.youtube.com/watch?v=3xKKI8irlQk)
+
+## Introduction
+The Demo application is a valuable tool to help you practice with requests. It’s used to create and test requests from the GeoServer. The user can see the response to the requests and solve the problems. To become familiar with the demo section, scroll down and click on the "Demo" link. This page contains these options:
```

Comment:
```suggestion
The Demo page provides valuable tool to help you practice with requests and understand how GeoServer works. The **Demo Requests** is used to create and test requests to GeoServer. The user can see the response to successful requests, or troubleshoot problems. To become familiar with the demo page, scroll down and click on the **Demo** link. This page contains these options:
```

Reply:
It is not so much a demo application as a page providing several tools to try out geoserver. the SRS list and other tools do not do requests at all, only demonstrate how geoserver works?

Reply:
Done.